feature-group-definitions/color-scheme.dist.yml

```diff
@@ -0,0 +1,22 @@
+# Generated from: color-scheme.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: color-scheme
+description: The `color-scheme` CSS property sets which color schemes (light or dark) an element uses and may prevent automatic dark mode adjustments by the browser.
+spec: https://drafts.csswg.org/css-color-adjust-1/#color-scheme-prop
+usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/637
+status:
+  baseline: low
+  baseline_low_date: 2022-02-03
```

Comment:
We need to review if this needs an override:
https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme#browser_compatibility